README.md

```diff
@@ -11,18 +11,18 @@ The framework source code can be found here: [cakephp/cakephp](https://github.co
 ## Installation
 
 1. Download [Composer](https://getcomposer.org/doc/00-intro.md) or update `composer self-update`.
-2. Run `php composer.phar create-project --prefer-dist --stability=dev cakephp/app [app_name]`.
+2. Run `php composer.phar create-project --prefer-dist cakephp/app:4.x [app_name]`.
```

Comment:
The `:4.x` isn't required in any command. With 4.0.0 tagged it will be used by default.